### PR TITLE
E2E tests for advanced gas input and priority fees

### PR DESCRIPTION
**Pull request by segun**
E2E tests for advanced gas fee input.

NOTE: RangeInput component is used to increase/descrese the gas fees and max priority fees and max base fees. I think the e2e for testing this component should be done in a separate PR.

### CODE REVIEW

File: e2e/specs/confirmations/advanced-gas-fees.spec.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    await TransactionConfirmationView.isMaxPriorityFeeCorrect('1.5');
+
+    await TransactionConfirmationView.tapAggressivePriorityGasOption();
+
+    await TransactionConfirmationView.isMaxPriorityFeeCorrect('2');
+
+    await TransactionConfirmationView.tapMaxPriorityFeeSaveButton();
+
+    await TransactionConfirmationView.isVisible();
```
Comment (cortisiko) [inline e2e/specs/confirmations/advanced-gas-fees.spec.js:70]: I would say don't just stop at updating the max priority fee. Submit the transaction! Maybe a bug might pop up after updating the priority then submitting the transaction

**Review by cortisiko — CHANGES_REQUESTED** [review]
small feedback.

**Comment by segun** [comment]
I have read the CLA Document and I hereby sign the CLA

File: e2e/specs/confirmations/advanced-gas-fees.spec.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import TransactionConfirmationView from '../../pages/TransactionConfirmView';
+
+import {
+  importWalletWithRecoveryPhrase,
+  switchToGoreliNetwork,
+} from '../../viewHelper';
+
+const VALID_ADDRESS = '0xebe6CcB6B55e1d094d9c58980Bc10Fed69932cAb';
+
+describe('Custom RPC Tests', () => {
```
[the remark this answers is not in the document]
Comment (segun) [reply e2e/specs/confirmations/advanced-gas-fees.spec.js:14]: Done

**Review by cortisiko — APPROVED** [review]
🌮 🌮 🌮

**Pull request merged**